src/Illuminate/Cache/FileStore.php

```diff
@@ -218,9 +218,11 @@ public function lock($name, $seconds = 0, $owner = null)
     {
         $this->ensureCacheDirectoryExists($this->lockDirectory ?? $this->directory);
 
+        $lockName = "illuminate:cache:lock:{$name}";
+
```

Comment:
The `illuminate:` prefix is generally reserved for internal keys.

Because this is a user space key, it feels weird to me to make it look internal. Certain monitoring systems, such as Pulse, Nightwatch, and Telescope would then start ignoring this key.

Perhaps, we suffix the key like we do with `:timer` to keep it in a user-land space.


```suggestion
        $lockName = "{$name}:file-store-lock";

```

Reply:
Appreciate it!

Reply:
Thank you!